portfolio/src/main/java/com/google/sps/Puzzle.java

```diff
@@ -0,0 +1,26 @@
+package com.google.sps;
+
+/** Puzzle domain object */
+public class Puzzle {
+
+  private String puzzleId; //Datastore generated id
+  private String image; // Blobstore url
+
```

Comment:
Add a constructor that can be used to initialize the fields:
```suggestion
Puzzle(String puzzleId, String imageUrl) {
  this.puzzleId = puzzleId;
  this.imageUrl = imageUrl.
}
```

Reply:
I've thought about it, I've added returns on the setters so in the future we can do something like:

```suggestion
Puzzle puzzle = new Puzzle()
  .setImageUrl("...")
  .setCreatorId("..."))
  .setDificulty(n)
  .setCreatorId("..."));
```
instead of creating multiple constructors for each combination.

Reply:
Ok that works to.